### DISCUSSION TITLE
USE_MERRA_CLIMO option

### DISCUSSION THREAD

**Discussion by SarahLu-NOAA** [ufs-community/ufs-srweather-app#1002] [Q&A] [answered]
Hi, UFS/SRW experts -

I'm runing SRW v2.2 over NE US for 2023 June period.  LBCs and ICs are taken from GFS.
I'd like to use MERRA2, instead of OPAC climo, in RRTMg.   My first attempt is to use FV3_GFS_v17_p8 suite and the results are way off (the tmp just tanks within 24 hours).     I roll back to FV3_GFS_v16 suite.  I modify my config.yaml as follow:
workflow: 
 FV3_NML_YAML_CONFIG_FN: /glade/work/clu/ufs/ufs-srweather-app/parm/FV3.merra.input.yml
task_run_fcst:
 USE_MERRA_CLIMO: true

In FV3.merra.input.yml, I set iaer to 1011 for the GFS_v16 suite.  The results are quite reasonable.  
fcst log file confirms GOCART climo is used.

While my approach works, I'm just wondering whether there is a better/smarter way to change namelist variable?   Thanks.

**Comment by gspetro-NOAA** [comment]
Hi @SarahLu-NOAA ,

The [USE_MERRA_CLIMO parameter](https://ufs-srweather-app.readthedocs.io/en/release-public-v2.2.0/CustomizingTheWorkflow/ConfigWorkflow.html#aerosol-climatology-parameter) generally should only be set to true with physics suites that support MERRA2 climatology and Thompson microphysics (i.e., FV3_GFS_v15_thompson_mynn_lam3km or FV3_GFS_v17_p8 physics suites). I don't believe the FV3_GFS_v16 physics suite was updated to support MERRA2 climatology and Thompson microphysics, either, so it is concerning that you were able to achieve reasonable results using a physics suite that doesn't support MERRA2 but encountered issues with a physics suite that should support it (although FV3_GFS_v17_p8 is not supported/tested with the SRW App, which could partially explain the issue). Regardless, these are interesting results, and I've reached out to some of our SMEs to investigate. 

Since FV3_GFS_v16 is not configured to use MERRA2 climatology, I believe you do need to make any desired changes manually in the YAML file, but the CCPP team generally cautions against using untested and unverified combinations of physics (this is part of why it is not easily configurable). That said, informed experimentation can lead to innovation! I'm hoping @mkavulich can weigh in with thoughts/suggestions, but regardless, it would be good to touch base with the [CCPP physics](https://github.com/NCAR/ccpp-physics/discussions) team, too. 

I will see if I can get an SME to follow up with you after the weekend with more information. 

Best,
Gillian Petro

**Reply by SarahLu-NOAA** [reply]
Many thanks, @gspetro-NOAA   I do reach out to the CCPP physics team on running GFS_v17_p8.   
The fcst log in my GFS_v16 experiment confirms that MERRA2, instead of OPAC, climo is used in RRTMg.  The results are also quite reasonable.  
I'm very surprised with my GFS_v17_p8 (very wrong) results and post an issue
https://github.com/ufs-community/ufs-srweather-app/issues/1004

After 72 hour integration with the same GFS LBCs/ICs, SRW v2.2 runs with v16 (out-of-the-box v16), v16x (v16 with MERRA2 climo) and rap (out-of-the-box RAP) suite have similar results.  But the one with v17 (out-of-the-box v17_p8) turns NE US to ice age.
![UFS_SRW_v2 2_f72](https://github.com/ufs-community/ufs-srweather-app/assets/56093533/d2b8bad4-c4eb-4ef0-94d7-ec5ab0e11671)

**Comment by gspetro-NOAA** [comment]
Hi @SarahLu-NOAA ! I know we have some developers taking a look at the related issue you posted (Issue #1004 ), so I will close this discussion and let @ulmononian and others continue to troubleshoot the issue there. :)

**Reply by SarahLu-NOAA** [reply]
Hi @gspetro-NOAA 
The EPIC and DTC colleagues are looking into related issues so this discuss can be closed.  Many thanks.

**Reply by gspetro-NOAA** [reply, marked as answer]
A bugfix addressing the contents of this discussion was added via PR #1055 .
